src/components/Icon/Icon.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import iconDefinitions from './iconDefinitions';
+import style from './Icon.scss';
+
+const Icon = (props) => {
+  const icon = iconDefinitions[props.icon];
+  const className = classnames(
+    props.className,
+    icon.svgClassName,
+    style.svg,
+  );
+
+  return (
+    <svg
+      className={className}
+      width={`${props.size}px`}
+      height={`${props.size}px`}
+      viewBox={icon.viewBox}
+      style={props.style}
```

Comment:
In all instances where `Icon` is used, I can only see `color` being set. Wouldn't it be more straightforward to simply have a `color` prop?

Reply:
Good one, yeah I can't think what other style we'd ever need to pass in.